testgrid/config/yaml2proto/testdata/IsExternalFalse.yaml

```diff
@@ -0,0 +1,32 @@
+default_test_group:
```

Comment:
Let's use a consistent naming pattern: other areas of this repo (and org) use hyphens to separate out words rather than camel case.

https://github.com/kubernetes/test-infra/tree/master/jenkins/job-configs/kubernetes-jenkins